docker/Dockerfile-x86_64

```diff
@@ -1,5 +1,5 @@
-FROM centos:5.11
-MAINTAINER The ManyLinux project
+FROM centos:6.8
```

Comment:
Since it deviates from the official recommendation, I think it warrants an explanation in the comments.